src/WPF/ArcGISRuntime.WPF.Viewer/Samples/GeometryEngine/Buffer/Buffer.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<UserControl x:Class="ArcGISRuntime.WPF.Samples.Buffer.Buffer"
+             x:Name="_this"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013"
+             xmlns:local="clr-namespace:ArcGISRuntime.WPF.Samples.Buffer">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <StackPanel  Orientation="Horizontal" Grid.Row="0">
```

Comment:
Per the new style convention, row and column need to be set before other properties (but after the name).